app/src/main/java/org/gnucash/android/ui/transaction/TransactionFormFragment.java

```diff
@@ -745,15 +741,15 @@ private void saveNewTransaction() {
 
         mAccountsDbAdapter.beginTransaction();
```

Comment:
This is confusing to me. Why the  mAccountsDbAdapter.beginTransaction() if we are adding records to mTransactionsDbAdapter?